### PR TITLE
NCD - Labs Results

### PR DESCRIPTION
**Pull request by anvmn**
#438

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-09-05 10-39

**Comment by anvmn** [comment]
@mariano-dagostino 

Please review.
Same as https://github.com/TIP-Global-Health/eheza-app/pull/452#issuecomment-1239705434, but here we're talking about Labs Results, So, in addition, we have dedicated encounter and activity pages.

File: client/src/elm/Pages/NCD/RecurrentActivity/Model.elm [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+--     }
+--
+--
+-- emptyNextStepsData : NextStepsData
+-- emptyNextStepsData =
+--     { referralForm = emptyReferralForm
+--     , medicationDistributionForm = emptyMedicationDistributionForm
+--     , healthEducationForm = emptyHealthEducationForm
+--     , activeTask = Nothing
+--     }
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/NCD/RecurrentActivity/Model.elm:96]: Remove or `@todo`?

File: client/src/elm/Pages/NCD/RecurrentActivity/Update.elm [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+update language currentDate id db msg model =
+    let
+        noChange =
+            ( model, Cmd.none, [] )
+
+        generateLabResultsMsgs nextTask =
+            Maybe.map (\task -> [ SetActiveLabResultsTask task ]) nextTask
+                |> Maybe.withDefault [ SetActivePage <| UserPage <| NCDRecurrentEncounterPage id ]
+
+        -- generateNextStepsMsgs personId nextTask =
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/NCD/RecurrentActivity/Update.elm:41]: Remove or `@todo`

File: client/src/elm/Pages/NCD/RecurrentActivity/Update.elm [hunk 3]
```diff
@@ -0,0 +1,564 @@
...
+                            )
+                        |> Maybe.withDefault []
+            in
+            ( model
+            , Cmd.none
+            , appMsgs
+            )
+                |> sequenceExtra (update language currentDate id db) extraMsgs
+
+        -- SetActiveNextStepsTask task ->
```
Comment (mariano-dagostino) [inline client/src/elm/Pages/NCD/RecurrentActivity/Update.elm:324]: Remove or `@todo`

**Review by mariano-dagostino — COMMENTED** [review]
@anvmn Looks good. a few comments regarding commented code.

File: client/src/elm/Pages/NCD/RecurrentActivity/Model.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Pages/NCD/RecurrentActivity/Model.elm:96]: This all has been implemented at #456 and #459 already.

File: client/src/elm/Pages/NCD/RecurrentActivity/Update.elm (hunk 2 above)
Comment (anvmn) [reply client/src/elm/Pages/NCD/RecurrentActivity/Update.elm:41]: This all has been implemented at #456 and #459 already.

File: client/src/elm/Pages/NCD/RecurrentActivity/Update.elm (hunk 3 above)
Comment (anvmn) [reply client/src/elm/Pages/NCD/RecurrentActivity/Update.elm:324]: This all has been implemented at #456 and #459 already.

**Pull request merged**